shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -95,14 +95,14 @@
     public static final String CACHE_GET_LATENCY = "segmentstore.cache.get_latency_ms";         // Histogram
     public static final String CACHE_READ_BYTES = "segmentstore.cache.read_bytes";              // Counter
     public static final String CACHE_TOTAL_SIZE_BYTES = "segmentstore.cache.size_bytes";        // Gauge
-    public static final String CACHE_GENERATION_SPREAD = "segmentstore.cache.gen";              // Histogram
+    public static final String CACHE_GENERATION_SPREAD = "cache.gen";              // Histogram
 
     // DurableDataLog (Tier1) stats
-    public static final String BK_TOTAL_WRITE_LATENCY = "segmentstore.bookkeeper.total_write_latency_ms";   // Including Queue. Per-container Histogram.
-    public static final String BK_WRITE_LATENCY = "segmentstore.bookkeeper.write_latency_ms";               // Exclusively the write to BK. Per-container Histogram.
+    public static final String BK_TOTAL_WRITE_LATENCY = "total_write_latency_ms";   // Including Queue. Per-container Histogram.
```

Comment:
Here (and below) you are removing `bookkeeper` too. Is the tag `bookkeeper` added by the histogram? Otherwise, we may confuse this with the write latency of the segment store.

Reply:
This is the ultimate change with the name, please let me know if there's still concern:
before: bookkeeper.segmentstore.bookkeeper.total_write_latency_ms
after: bookkeeper.write_latency_ms.